src/test/scopes/jsx/__snapshots__/jsx-scope.e2e.test.ts.snap

```diff
@@ -1,5 +1,78 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
+exports[`class: JsxScope > findLocalPackages > correctly finds local packages 1`] = `
```

Comment:
snapshpot needs to be updated

Reply:
Done! Apparently my test:watch command and hitting `u` didn't remove the obsolete ones. Should be all good now though.